src/remote-desktop.c

```diff
@@ -124,6 +128,26 @@ remote_desktop_session_can_select_sources (RemoteDesktopSession *session)
   g_assert_not_reached ();
 }
 
+gboolean remote_desktop_session_can_enable_clipboard(
+    RemoteDesktopSession *session) {
```

Comment:
Seems most of the changes to this file needs to be changed to follow the existing coding style.